aldryn_config.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from aldryn_client import forms
+
+
+class Form(forms.BaseForm):
+    editor_theme = forms.CharField(
+        'Custom editor theme, (e.g. "github", default: "twilight")',
+        required=False,
+    )
+    editor_mode = forms.CharField(
+        'Custom editor mode (e.g. "javascript", default: "html")',
+        required=False,
+    )
+    enable_search = forms.BooleanField(
```

Comment:
yeah my thought being to expose all settings in general in the CP. They might not be essentially developers though need to be aware of what they might activate. Otherwise we prohibit certain features for users.